May add the ability to renew a previously created platform.. or not !

croftophile opened the pull request.

croftophile committed:
May add the ability to renew a previously created platform.. or not !

benemart commented on deployment/roles/init_bdd/templates/iam/03_application_ref.js.j2:
```diff
@@ -1,6 +1,8 @@
 db.applications.remove({});
+{% endif %}
 
 db.applications.insert({
```
Il faut que le bloc de création des applications soit aussi dans le renew, sinon les apps sont recrées avec un _id différent et apparaissent en double dans le portail

benemart reviewed (changes requested):
Dans la collection applications (base iam), il faut inclure la créations des applications dans le bloc renew_platform pour éviter qu'elles soient dupliquées (_id random). J'ai eu le cas lors de mon déploiement

croftophile committed:
Update 03_application_ref.js.j2

Added one more conditional to bypass apps duplicates.
Better way may be to manage some kind of "insert or update", if new apps are added later.
